templates/applications/application.html

```diff
@@ -4,136 +4,266 @@
-                    {% endif %}
-                    Late stage                    
-                </h4>
-                <p>Once you have successfully completed your early stage interviews you’ll be invited to meet with senior leads within your management chain</p>
-            </li>
-            <li class="p-list_item">
-                <h4 class='u-no-margin--bottom'>
-                    {% if "stage_progress" in application and application["stage_progress"]["offer"] %}
-                    <i class="p-icon--success"></i> 
-                    {% else %}<i class="p-icon--information"></i> 
-                    {% endif %}
-                    Offer
-                </h4>                
-            </li>
-        </ol>
+  </section>
+  <section class="p-strip">
+    <div class="u-fixed-width">
+      <h2 class="p-heading--4">Your application process</h2>
+      <p class="u-sv2">Once your application is accepted, you will see more details about its status. <a>See what our CEO says about our hiring process</a>.</p>
     </div>
-    <div class="row">
-        <h2>Life at Canonical</h2>
+    <div class="u-fixed-width">  
+      <ol class="p-stepped-list--detailed p-list--divided">
+        <li class="p-stepped-list__item">
+          <h3 class='p-process-stepped-list__title u-no-margin--bottom'>
+            {% if "stage_progress" in application and application["stage_progress"]["assessment"] %}<span class="p-on-pass"><i class="p-icon--success"></i> </span>
+            {% elif "stage_progress" in application and not application["stage_progress"]["assessment"] and application["stage_progress"]["application"] %}
+            <span class="p-on-stage"></span> 
+            {% else %}<span class="p-not-on-stage"></span> 
+            {% endif %}
+            Application review
+          </h3>
+          <p class="p-stepped-list__content">An experienced Hiring Lead from our business will review your CV submission against the job criteria set for this role.</p>
+        </li>
+        <li class="p-stepped-list__item">
+          <h3 class='p-process-stepped-list__title u-no-margin--bottom'>
+            {% if "stage_progress" in application and application["stage_progress"]["early_stage"] %}<span class="p-on-pass"><i class="p-icon--success"></i> </span>
+            {% elif "stage_progress" in application and not application["stage_progress"]["early_stage"] and application["stage_progress"]["assessment"] %}
+            <span class="p-on-stage"></span> 
+            {% else %}<span class="p-not-on-stage"></span> 
+            {% endif %}
+            Assessment
+          </h3>
+          <div class="p-stepped-list__content">
+            {% if "stage_progress" in application and application["stage_progress"]["early_stage"] %}
+            <p>There are three parts to this stage:</p>
+            <p onclick="showProgressDetail('assessment')" class="show-more-assessment">Show more</p>
```

Comment:
These links are not accessible via keyboards. Could you use a link with `href=""` and `e.preventDefault();` please?